### PR TITLE
:memo: docs: add documentation for an existing Airflow stack (#9)

### PR DESCRIPTION
**Pull request by mostafaghadimi**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -61,10 +62,46 @@ Deploying the project is straightforward:
 
 4. **Enjoy**: That’s it! Your Apache Airflow monitoring stack is now operational.
 
+## Integration Steps for Airflow Monitoring Stack in Existing Setup :speech_balloon:
+
+To integrate the monitoring stack for Apache Airflow into an existing setup with Airflow and Prometheus, follow these instructions:
+
+1. **Install the required components**: Make sure you have Airflow and Prometheus ans Grafana installed and configured in your environment.
```
Comment (niyushabaghayi) [inline README.md:69]: This line `Prometheus <ans> Grafana` , please change <ans> to <and>.

File: README.md [hunk 2]
```diff
@@ -61,10 +62,46 @@ Deploying the project is straightforward:
...
+
+3. **Modify Airflow configuration**: Open your Airflow configuration file (airflow.cfg) and add the following lines:
+
+   ```bash
+   AIRFLOW__METRICS__STATSD_ON: "True"
+   AIRFLOW__METRICS__STATSD_HOST: "statsd-exporter"
+   AIRFLOW__METRICS__STATSD_PORT: "9125"
+   AIRFLOW__METRICS__STATSD_PREFIX: "airflow"
+
+   Adjust the values based on your specific configuration.
```
Comment (niyushabaghayi) [inline README.md:81]: If we can have this line:
```
Adjust the values based on your specific configuration.
```
like this:

:alert: Adjust the values based on your specific configuration.

and putting it outside the code is better.

File: README.md [hunk 3]
```diff
@@ -61,10 +62,46 @@ Deploying the project is straightforward:
...
+4. **Configure Prometheus**: In your Prometheus configuration file (prometheus.yml), add the scrape configuration for the StatsD exporter:
+
+    ```bash
+    - job_name: 'statsd-exporter'
+      scrape_interval: 15s
+      static_configs:
+        - targets:
+          - 'statsd-exporter:9102'
+
+    Save the file after making the changes.
```
Comment (niyushabaghayi) [inline README.md:93]: Also about this line like line `81`.

File: README.md [hunk 4]
```diff
@@ -61,10 +62,46 @@ Deploying the project is straightforward:
...
+    ```
+
+5. **Copy Grafana Dashboard JSON file**: From the `airflow-monitoring-and-alerting` repository, navigate to the config_files/grafana/var/lib/grafana/dashboards directory. Copy the JSON files of the desired dashboards you want to add to your Grafana instance.
```
Comment (niyushabaghayi) [inline README.md:96]: please put `config_files/grafana/var/lib/grafana/dashboards` in ``.

File: README.md [hunk 5]
```diff
@@ -61,10 +62,46 @@ Deploying the project is straightforward:
...
+*Note: The specific directory path may vary depending on your Grafana setup.*
```
Comment (niyushabaghayi) [inline README.md:97]: Please put this Note line in a separate line (one \n) and not just attach the previous lines.

File: README.md [hunk 6]
```diff
@@ -61,10 +62,46 @@ Deploying the project is straightforward:
...
+We welcome contributions to this repository! If you’re interested in contributing, please take a look at our [CONTIRIBUTION.md](https://github.com/data-burst/airflow-monitoring-and-alerting/blob/master/CONTRIBUTING.md) file for more information on how to get started. We look forward to collaborating with you!
 
 ## License 📄
 
-This repository is licensed under the MIT License, which is a permissive open-source license that allows for reuse and modification of the code with few restrictions. You can find the full text of the license in [this](https://github.com/data-burst/airflow_monitoring_and_alerting/license) file.
+This repository is licensed under the MIT License, which is a permissive open-source license that allows for reuse and modification of the code with few restrictions. You can find the full text of the license in [this](https://github.com/data-burst/airflow-monitoring-and-alerting/license) file.
```
Comment (niyushabaghayi) [inline README.md:107]: It appears that the link is corrupted. It is possible to link them relatively. (and also about CONTRIBUTING.md file)

**Review by niyushabaghayi — CHANGES_REQUESTED** [review]
Thanks @iamdorsa ! :purple_heart:

File: README.md (hunk 6 above)
Comment (mostafaghadimi) [reply README.md:107]: No, it's not corrupted. You are failed to see it, since it is not merged to the main branch yet!

**Review by niyushabaghayi — APPROVED** [review]

**Review by mostafaghadimi — COMMENTED** [review]
Nice job!

**Pull request merged**